### PR TITLE
Feature/bc 94 color clean up

### PR DESCRIPTION
**Pull request by monaweitzenberg**
# Short Description

clean up the colors in the old client and nuxt and fixing small UX issues

## Links to Ticket and related Pull-Requests

https://ticketsystem.hpi-schul-cloud.org/browse/BC-94

## Changes
cleaning primary, secondary and accent color; delete tertiary color

## Screenshots of UI changes
dBildungsCloud
open
default

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/themes/brb/styles/custom-styles.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // change the logo background to secondary color as in the legacy client
 .sidebar .top-sidebar .logo {
-	background-color: var(--color-secondary);
+	// background-color: var(--color-primary);
```
Comment (pkleybolte) [inline src/themes/brb/styles/custom-styles.scss:3]: better remove than comment

File: src/themes/demo/styles/variables/_colors.scss [hunk 2]
```diff
@@ -4,14 +4,15 @@ $darken: 25%;
 :root {
 	/* main colors */
 	--color-primary: #b10438;
+	--color-primary: #b1063a;
```
Comment (pkleybolte) [inline src/themes/demo/styles/variables/_colors.scss:7]: primary defined twice

File: src/themes/demo/styles/variables/_colors.scss [hunk 3]
```diff
@@ -4,14 +4,15 @@ $darken: 25%;
...
-	--color-secondary: #e2661d;
-	--color-secondary-light: color-mod(var(--color-secondary) tint(#{$brighten}));
-	--color-secondary-dark: color-mod(var(--color-secondary) shade(#{$darken}));
-	--color-tertiary: #455b6a;
-	--color-tertiary-light: #617480;
-	--color-tertiary-dark: #344450;
+	--color-secondary: #455b6a;
+	--color-secondary-light: #617480;
+	--color-secondary-dark: #344450;
+	--color-accent: #e98404;
```
Comment (pkleybolte) [inline src/themes/demo/styles/variables/_colors.scss:13]: Is this the right color?

File: src/themes/int/styles/variables/_colors.scss [hunk 4]
```diff
@@ -3,14 +3,9 @@ $brighten: 90%;
...
-	--color-primary-light: #417491;
-	--color-primary-dark: #002239;
-	--color-secondary: #d9174c;
-	--color-secondary-light: #f15574;
-	--color-secondary-dark: #a31139;
-	--color-accent: #74c599;
-	--color-gray-dark: #17212a;
-	--color-gray-light: color-mod(var(--color-gray-dark) tint(#{$brighten}));
-	--color-sidebar-menu-item-active: var(--color-primary-dark);
+	--color-primary: #0a688c;
```
Comment (pkleybolte) [inline src/themes/int/styles/variables/_colors.scss:6]: color correct?

File: src/themes/n21/vuetify.options.js [hunk 5]
```diff
@@ -6,9 +6,9 @@ const custom = {
 		dark: false,
 		themes: {
 			light: {
-				primary: "#78aae5",
-				secondary: "#b10438",
-				accent: "#455b6a",
+				primary: "#6E9CD2",
```
Comment (pkleybolte) [inline src/themes/n21/vuetify.options.js:9]: color  correct? different color in _colors.scss

File: src/themes/int/styles/variables/_colors.scss (hunk 4 above)
Comment (pkleybolte) [reply src/themes/int/styles/variables/_colors.scss:6]: Change in vuetify options necessary?

File: src/themes/demo/styles/variables/_colors.scss (hunk 2 above)
Comment (pkleybolte) [reply src/themes/demo/styles/variables/_colors.scss:7]: change in vuetify options necessary?

File: src/themes/brb/styles/variables/_colors.scss [hunk 6]
```diff
@@ -3,10 +3,13 @@ $darken: 25%;
 
 :root {
 	/* main colors */
-	--color-primary: #449494;
-	--color-secondary: #e4032e;
+	--color-primary: #0a9396;
```
Comment (pkleybolte) [inline src/themes/brb/styles/variables/_colors.scss:6]: change in vuetify options necessary?

File: src/themes/base/styles/variables/_colors.scss [hunk 7]
```diff
@@ -6,12 +6,12 @@ $darken: 25%;
 	--color-primary: #b1063a;
 	--color-primary-light: color-mod(var(--color-primary) tint(#{$brighten}));
 	--color-primary-dark: color-mod(var(--color-primary) shade(#{$darken}));
-	--color-secondary: #e98404;
-	--color-secondary-light: color-mod(var(--color-secondary) tint(#{$brighten}));
-	--color-secondary-dark: color-mod(var(--color-secondary) shade(#{$darken}));
-	--color-tertiary: #455b6a;
-	--color-tertiary-light: #617480;
-	--color-tertiary-dark: #344450;
+	--color-secondary: #455b6a;
```
Comment (pkleybolte) [inline src/themes/base/styles/variables/_colors.scss:9]: change in vuetify options necessary?

File: src/themes/open/styles/variables/_colors.scss [hunk 8]
```diff
@@ -3,10 +3,10 @@ $darken: 25%;
 
 :root {
 	/* main colors */
-	--color-primary: #126dc4;
-	--color-secondary: #df0b40;
+	--color-accent: #126dc4;
+	--color-primary: #df0b40;
```
Comment (pkleybolte) [inline src/themes/open/styles/variables/_colors.scss:7]: no secondary color defined
change in vuetify options necessary?

File: src/themes/thr/styles/variables/_colors.scss [hunk 9]
```diff
@@ -1,18 +1,10 @@
...
-	--color-primary-dark: #002f5b;
-	--color-secondary: #f56b00;
-	--color-secondary-light: #ff9c3f;
-	--color-secondary-dark: #bb3b00;
-	--logo-gradient: linear-gradient(
-		45deg,
-		var(--color-primary),
-		var(--color-primary-light)
-	);
+	--color-primary: #2876d0;
```
Comment (pkleybolte) [inline src/themes/thr/styles/variables/_colors.scss:4]: change in vuetify options necessary?

**Review by pkleybolte — CHANGES_REQUESTED** [review]

File: src/themes/brb/styles/custom-styles.scss (hunk 1 above)
Comment (monaweitzenberg) [reply src/themes/brb/styles/custom-styles.scss:3]: Da dann die Datei keinen Sinn mehr macht, habe ich sie gleich gelöscht. :)

File: src/themes/demo/styles/variables/_colors.scss (hunk 2 above)
Comment (monaweitzenberg) [reply src/themes/demo/styles/variables/_colors.scss:7]: theme demo hat keine vuetify options Datei. Soll das so?

File: src/themes/demo/styles/variables/_colors.scss (hunk 3 above)
Comment (monaweitzenberg) [reply src/themes/demo/styles/variables/_colors.scss:13]: yes, it is.

File: src/themes/demo/styles/variables/_colors.scss [hunk 10]
```diff
@@ -6,12 +6,12 @@ $darken: 25%;
 	--color-primary: #b10438;
```
Comment (pkleybolte) [inline src/themes/demo/styles/variables/_colors.scss:6]: Demo uses the vuetify.options.js of base, but base has a different primary. We need a vuetify.options.js to have a consistent primary color in vuetify and non-vuetify components.

**Review by pkleybolte — CHANGES_REQUESTED** [review]

File: src/themes/base/styles/variables/_colors.scss (hunk 7 above)
Comment (monaweitzenberg) [reply src/themes/base/styles/variables/_colors.scss:9]: There was nothing to adjust.

File: src/themes/brb/styles/variables/_colors.scss (hunk 6 above)
Comment (monaweitzenberg) [reply src/themes/brb/styles/variables/_colors.scss:6]: The vuetify options are now also adjusted.

File: src/themes/int/styles/variables/_colors.scss (hunk 4 above)
Comment (monaweitzenberg) [reply src/themes/int/styles/variables/_colors.scss:6]: The vuetify options are now also adjusted. And colors are correct.

File: src/themes/thr/styles/variables/_colors.scss (hunk 9 above)
Comment (monaweitzenberg) [reply src/themes/thr/styles/variables/_colors.scss:4]: The vuetify options are now also adjusted.

**Comment by luizzaly** [comment]
Bildschirmfoto 2021-09-06 um 16 17 49

Bei aktuellen Aufgaben in "open" scheint noch das falsche Logo gezogen zu werden..

**Review by luizzaly — APPROVED** [review]
looks good! 👍🏼

**Review by pkleybolte — APPROVED** [review]

**Comment by monaweitzenberg** [comment]
@schul-cloud-bot update screenshots

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**